src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
```

Comment:
This has no merit. There could be any number of other means to upgrade the firmware.
This will remain the case so long as interfaces don't disclose the device or firmware version at all, or truthfully, which is to say for the foreseeable future..